src/utils.js

```diff
@@ -69,6 +69,30 @@ export const setQueryParam = (param, value) => {
   localStorage.savedQueryParams = qs.toString()
 }
 
+export const appendQueryParam = (param, value) => {
+  const qs = new URLSearchParams(window.location.search)
+  if (qs.get(param) && value) {
```

Comment:
Any reason for this if statement? From what I can see it still works perfectly fine with just L75.

Reply:
Just to copy the discord discussion here for future reference: removing this `if` gives an extra comma in front of the query string from the first append.

Reply:
As discussed in discord, we don't need to check for empty value.